aiolifx/fixtures/fixtures.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+from aiolifx.fixtures.chain_light import ChainLightMixin
+from aiolifx.fixtures.color_light import ColorLightMixin
+from aiolifx.fixtures.hev_light import HevLightMixin
+from aiolifx.fixtures.light import LightMixin
+from aiolifx.fixtures.multizone_light import MultizoneLightMixin
+from aiolifx.fixtures.switch import SwitchMixin
+from aiolifx.fixtures.matrix_light import MatrixLightMixin
+from aiolifx.products import products_dict
+
+
+class Light(LightMixin):
+    pass
+
+class ColorLight(LightMixin, ColorLightMixin):
+    pass
+
+class MultizoneLight(LightMixin, ColorLightMixin, MultizoneLightMixin):
+    pass
+
+class HevLight(LightMixin, HevLightMixin):
+    pass
+
+class ChainLight(LightMixin, MatrixLightMixin, ChainLightMixin):
+    pass
+
+class MatrixLight(LightMixin, MatrixLightMixin):
+    pass
+
+class Switch(SwitchMixin):
+    pass
+
+
+def get_fixture(product_id, req_with_resp, req_with_ack, fire_and_forget):
+    product = products_dict[product_id]
+    if product.relays and product.buttons:
+        return Switch(req_with_resp, req_with_ack, fire_and_forget)
+    if product.multizone and product.extended_multizone:
+        return MultizoneLight(req_with_resp, req_with_ack, fire_and_forget)
```

Comment:
All extended multizone lights support all multizone requests, but not all multizone lights support the extended multizone requests, i.e. extended multizone is an extension of multizone.